README.md

```
@@ -277,27 +277,29 @@ lib/
 | 📦 **Storage** | Upload & access videos, notes, documents |
 | 🗄️ **Database** | Full CRUD: Notes, MCQs, Progress tracking |
 | 💻 **Dart Examples** | Code snippets with `http` and `supabase_flutter` |
+---
+# Installation and Usage Guide
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)